platforms/dotnet/versions/bookworm/versionsToBuild.txt

```diff
@@ -14,3 +14,4 @@
 8.0.100-preview.7.23376.3, 8bc71a586382f0e264024707f2f3af9a2f675dd5d4fbdd4bced7ab207141fb74d7c6492dfd94373505962b8597ae379259d152e4ace93a65dad0f89600afecd8,
 8.0.100-rc.1.23463.5, ac941fd16fd7c328f7cc44b132b4253ddb2b6a6c152af5f43c71c6cd0d468c89b7276ebf6c08895dcb6e5e25f7cae83b6fbacb91cfcc4a61d49b5657a834a901,
 8.0.100-rc.2.23502.2, 45f09e7b031f4cf5b4dcead240fe47e2e3731d97d22aa96d3a02a087322658606cc22792053c3784c44f15d7c9bad0ac9dbda90def7b4e197f2955dca9a5bb6c,
```

Comment:
I merged with `williamhe/update-dotnet-8-sdk` to make sure there was no merge conflicts, after `williamhe/update-dotnet-8-sdk` is merged.